doc/en/user/source/data/cascaded/wms.rst

```diff
@@ -1,78 +1,76 @@
-.. _data_external_wms:

```

Comment:
> I just did an initial review with some general remarks, didn't cover all files. I tried this manually but it doesn't seems to work.
> I setup two GeoServer instance, 8060 and 8061, the second one contains the usual GeoServer release test data, and I configured a remote store for the second one using WMS 1.3.0 capabilities document. I published the remote layers on the first instance under the it.geosolutions workspace.
> Try to fetch legend for the default style using WMS 1.3.0, produced the following result:
> http://localhost:8060/geoserver/ows?service=WMS&version=1.3.0&request=GetLegendGraphic&format=image/png&width=20&height=20&layer=it.geosolutions:tiger_roads
> 
> <?xml version="1.0" encoding="UTF-8"?><ServiceExceptionReport version="1.3.0" xmlns="http://www.opengis.net/ogc" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.opengis.net/ogc http://localhost:8060/geoserver/schemas/wms/1.3.0/exceptions_1_3_0.xsd">   <ServiceException>
>       Can&apos;t obtain the schema for the required layer.
> Server does not specify a GetLegendGraphic operation. Cannot be performed
> </ServiceException></ServiceExceptionReport>
> ... using WMS 1.1.1:
> http://localhost:8060/geoserver/ows?service=wms&version=1.1.1&request=GetLegendGraphic&format=image/png&width=20&height=20&layer=it.geosolutions:tiger_roads
> 
> <?xml version="1.0" encoding="UTF-8" standalone="no"?><!DOCTYPE ServiceExceptionReport SYSTEM "http://localhost:8060/geoserver/schemas/wms/1.1.1/WMS_exception_1_1_1.dtd"> <ServiceExceptionReport version="1.1.1" >   <ServiceException>
>       Can&apos;t obtain the schema for the required layer.
> Server does not specify a GetLegendGraphic operation. Cannot be performed
> </ServiceException></ServiceExceptionReport>
> Can you give me some inputs on this, maybe I'm missing something  😨.

Can you please try this

http://cloudsdi.geo-solutions.it/geoserver/wms?service=wms&version=1.1.1&request=GetCapabilities

Reply:
Wondering if the client ends up using version 1.3.0 for some reason (e.g., negotiation happening regardless of the version number specified in the caps url)), since in 1.3.0 there is no indication of GetLegendGraphic in the wms 1.3.0 caps (formally because we do not support SLD 1.1 extensions, but only SLD 1.0 extensions, and GetLegendGraphic is one of them.... we should check if the sld 1.1/wms 1.3.0 version has anything substantially different)

Reply:
This has been adressed by converting a getMap request of remote WMS service and turning it into a GetLegend Graphic. Now 1.3.0 version is also supported